modeline/disk/disk.asd

```diff
@@ -1,11 +1,12 @@
 ;;;; disk.asd
 
-(asdf:defsystem #:disk
+(asdf:defsystem :disk
   :serial t
   :description "Display filesystem information in the modeline"
   :author "Morgan Veyret"
   :license "GPLv3"
   :depends-on (:stumpwm
-               :cl-diskspace)
+               :cl-diskspace
+               #+linux :cl-mount-info)
```

Comment:
```suggestion
               (:feature :linux "cl-mount-info"))
```

https://github.com/fare/asdf/blob/master/doc/best_practices.md#conditional-code

Reply:
Hi!

Interesting! Thank you to point out this! :)

Bye!
C.